src/applications/vaos/new-appointment/components/VAFacilityPage/EligibilityModal.jsx

```diff
@@ -20,18 +20,24 @@ export default function EligibilityModal({
   return (
     <VaModal
       id="eligibilityModal"
-      status="warning"
+      status={status}
       visible
       onCloseEvent={onClose}
-      hideCloseButton={status === FETCH_STATUS.loading}
       modalTitle={title}
       ariaLabel={title}
       data-testid="eligibilityModal"
       role="alertdialog"
       uswds
     >
       <div aria-atomic="true" aria-live="assertive">
-        {content}
+        <p>{content}</p>
+        <p>
```

Comment:
Is it safe to add this content in this component? It would make sense if this modal is only used for 502 errors or if all eligibility messages share this content. Going off of the Figma file from design I'm having trouble parsing if either of these cases are true so can you confirm why we can add this content here instead of `getEligibilityMessage`?

Reply:
@JunTaoLuo - Added here due to not being able to add html formatting to the content string. The html formatting displays as part of the string. I did check and the EligibilityModal is only used by the 'VAFacilityPageV2' component right now. The added message seems generic enough to apply to any error message.

Reply:
Hmm if we are adding something to the error message, shouldn't we run that by the design folks? I see that it is only used by `VAFacilityPageV2` but what I'm concerned about is if it is used to display other types of errors, some of which shouldn't have the additional content.

Reply:
Hmm so I tried the scenario where we facility eligibility fails due to `facility-request-limit-exceeded` and I see the new content appended to the end of the warning:
<img width="499" alt="image" src="https://github.com/user-attachments/assets/bbb57874-0e91-4a27-9f76-df1cffa7c502" />
Compared to what's in the Figma files:
<img width="362" alt="image" src="https://github.com/user-attachments/assets/c8e6b1a4-36a6-4792-adba-49dbbda7fc9f" />
Can we just make the new content conditional on the `status` variable and only display it if it is `== 'error'`?

Reply:
Yes. Thanks.